bin/rpm-s3

```diff
@@ -105,6 +105,14 @@ class FileGrabber(object):
         os.symlink(realfilename, filename)
         return filename
 
+
+def getclient(base):
+    if os.getenv('AWS_ACCESS_KEY'):
+        return boto.connect_s3(os.getenv('AWS_ACCESS_KEY'), os.getenv('AWS_SECRET_KEY').get_bucket(base.netloc))
```

Comment:
Just saw this.  Sorry all.

On Sun, May 31, 2015 at 2:05 PM, Benjamin Eberlei <notifications@github.com>
wrote:

> In bin/rpm-s3
> <https://github.com/crohr/rpm-s3/pull/2#discussion_r31392785>:
>
> > @@ -105,6 +105,14 @@ class FileGrabber(object):
> >          os.symlink(realfilename, filename)
> >          return filename
> >
> > +
> > +def getclient(base):
> > +    if os.getenv('AWS_ACCESS_KEY'):
> > +        return boto.connect_s3(os.getenv('AWS_ACCESS_KEY'), os.getenv('AWS_SECRET_KEY').get_bucket(base.netloc))
>
> Introduced a bug, the parenthesis are wrong, see #3
> <https://github.com/crohr/rpm-s3/issues/3>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/crohr/rpm-s3/pull/2/files#r31392785>.
>